tap_hubspot/hubspot.py

```diff
@@ -173,9 +173,27 @@ def streams(self, start_date: datetime, end_date: datetime):
             yield from self.get_submissions()
         elif self.tap_stream_id == "contacts_events":
             yield from self.get_contacts_events()
+        elif self.tap_stream_id == "deal_properties":
+            yield from self.get_properties("deals")
+        elif self.tap_stream_id == "contact_properties":
+            yield from self.get_properties("contacts")
+        elif self.tap_stream_id == "company_properties":
+            yield from self.get_properties("companies")
         else:
             raise NotImplementedError(f"unknown stream_id: {self.tap_stream_id}")
```

Comment:
If we upgraded to [Python 3.10](https://docs.python.org/3.10/whatsnew/3.10.html), this could be turned into a somewhat more sexy `match` statement.

Reply:
very cool!